SoftLayer/CLI/core.py

```diff
@@ -103,7 +125,8 @@ def get_command(self, ctx, name):
               is_flag=True,
               required=False,
               help="Use demo data instead of actually making API calls")
-@click.version_option(prog_name="slcli (SoftLayer Command-line)")
+@click.version_option(prog_name="slcli (SoftLayer Command-line)",
```

Comment:
Doing this will allow us to only query for the latest version only when someone actually runs `--version`. 
```
@click.option('--version', is_flag=True, expose_value=False, is_eager=True, callback=get_version,
              help="Show version information.")
```